packages/components/src/external-link/index.tsx

```diff
@@ -65,18 +63,16 @@ function UnforwardedExternalLink(
 			target="_blank"
 			rel={ optimizedRel }
 			ref={ ref }
+			style={ { textDecoration: 'none' } }
 		>
-			{ children }
+			<span style={ { textDecoration: 'underline' } }>{ children }</span>
 			<VisuallyHidden as="span">
 				{
 					/* translators: accessibility text */
 					__( '(opens in a new tab)' )
 				}
 			</VisuallyHidden>
-			<StyledIcon
-				icon={ external }
-				className="components-external-link__icon"
-			/>
+			&nbsp;&#8599;
```

Comment:
I'd advise against spacing with `&nbsp;` because it makes it way more difficult to define equal spacing across all browsers and to precisely define the margin we want to have on the left side of the icon. In general, using content features for styling purposes is discouraged for the same reasons.